core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/TokenFactory.java

```diff
@@ -15,26 +15,23 @@
  */
 package com.datastax.oss.driver.internal.core.metadata.token;
 
+import com.datastax.oss.driver.api.core.metadata.token.Partitioner;
 import com.datastax.oss.driver.api.core.metadata.token.Token;
 import com.datastax.oss.driver.api.core.metadata.token.TokenRange;
-import java.nio.ByteBuffer;
```

Comment:
Please don't do unrelated cleanups.

Reply:
Again, this is caused by automatic style checker:
```
[ERROR] Found 1 non-complying files, failing build
[ERROR] To fix formatting errors, run "mvn com.coveo:fmt-maven-plugin:format"
```
(after adding back this import)

Reply:
ok.